workbench-core/accounts/src/services/costCenterService.test.ts

```diff
@@ -253,9 +252,9 @@ describe('CostCenterService', () => {
               removeUndefinedValues: true
             })
           });
-          accountId = `${resourceTypeToKey.account.toLowerCase()}-sampleAccId`;
+          accountId = `acc-${randomUuid}`;
```

Comment:
if this is used in multiple places and known ahead of time, lets make it a const instead of constructing it at each place its needed.